content/Discussions/2019-12-31-Rawski_Marr.md

```diff
@@ -0,0 +1,189 @@
+Read his BBS target article on it [@brette2019], or his earlier "Philosophy of the spike" [@brette2015].
+
+
+### Relationships matter more
+
+My second gripe is that the three levels make little sense on their own, but the linguist argument behaves as though they do.
+I think this is mostly Marr's fault, for saying things like "these three levels are coupled, but only loosely".
+A big motivation for Marr was to clearly delineate computational and algorithmic/representational explanations.
+It's obvious why.
+Numerous algorithmic models are compatible with a given computational problem.
+Explanation via algorithm is also useless unless you know what problem it solves, with which resources, and with what guarantees.
+
+Here's the problem, though, put clearly by Gyorgi Buzsaki in [his new book](https://global.oup.com/academic/product/the-brain-from-inside-out-9780190905385?cc=us&lang=en&).
+While numerous algorithmic models are compatible with a given computational problem, there's evidence that only one or a few are compatible with those implemented by neuronal circuits.
+For example, in invertebrate neural circuits, like the crab somatogastric ganglion, there are only a handful of neurons, each of them different.
+Even though we've got the circuit wiring figured out, we don't know the mechanisms that let the cell-to-cell interactions enable the sequential spatiotemporal patterns generating rhythmic output [@harris1992; @marder2015].
+The degeneracy of this circuit is very rich, with hundreds of circuit solutions to the same rhythm in related species and within the members of the same species [@prinz2004].
+Buzsaki says that because of this problem, "it is hard to find an impactful neuroscience experiment which is based on Marr’s suggested three-stage strategy".
+
+Yikes.
+
+Later in the book, Marr is more expressive about explanatory relationships.
+Take a look at his diagram below.
+It's hard to draw a three-way split there that resembles the Three Levels in a coherent way.
+Also notice that representation, which before was confined to the middle level as an object, now has its own flow.
+This makes a bit more sense under the representation-as-process view. However, I have an even harder time trying to fit modern linguistics into one level here. 
+
+![more levels]({static}/img/Rawski/Marr/7Lvls.jpg){ width=60% }
+
+In his [afterward to the re-release of the book](http://cbcl.mit.edu/publications/ps/marr-vision-afterword.pdf), Tommaso Poggio is critical of level independence.
+He recounts the gritty origin story of the Three Levels: while they were colleagues, Marr adapted the framework of @reichardt1976, which distinguished behavioral, algorithmic, and implementational levels (Marr swapped out 'computation' for behavior).
+Poggio notes that "one key aspect of the original argument disappeared in the process".
+What was it?
+"It is necessary to study nervous systems at all levels simultaneously".
+Great advice.  
```

Comment:
There might be trailing spaces here, there's a sudden linebreak in the typeset article.